dataland-frontend/src/components/general/TheCampaigns.vue

```diff
@@ -0,0 +1,166 @@
+<template>
```

Comment:
This component is currently commented out on the Landing Page.  Why?

Reply:
This component won't go live on this first version but it will on the next one. I removed it for now

Reply:
Ok sounds good.  But for me it is still there when I pull. Also in the remote github repo:

![image](https://github.com/d-fine/Dataland/assets/99255296/05a50b00-5454-47f3-ae2e-5ddd05467aac)